pythonforandroid/recipe.py

```diff
@@ -799,19 +821,6 @@ def install_python_package(self, arch, name=None, env=None, is_dir=True):
             elif self.call_hostpython_via_targetpython:
```

Comment:
Wow @inclement, you are right, I think that we should discuss about if we should fix the python2legacy recipe, because if we fix it, we will need those lines, otherwise I think that we could remove it and probably we could remove all the references to call_hostpython..., but this will touch a lot of files...
About python2legacy recipe, I already said in discord that I have it done in another branch, so it's only matter to decide if we safe it or remove it...so...

what do you think @AndreMiras, @inclement , @tito and @KeyWeeUsr,

**Should we keep the python2legacy recipe?**